packages/esm-patient-test-results-app/src/overview/common-datatable.component.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  DataTable,
+  Table,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableCell,
+  TableBody,
+} from 'carbon-components-react';
+import { TypedTableRow } from './helpers';
+import { OverviewPanelData } from '../overview/useOverviewData';
+import styles from './common-datatable.scss';
+
+interface CommonDataTableProps {
+  data: Array<OverviewPanelData>;
+  tableHeaders: Array<{
+    key: string;
+    header: string;
+  }>;
+  title?: string;
+  toolbar?: React.ReactNode;
+  description?: React.ReactNode;
+}
+
+const CommonDataTable: React.FC<CommonDataTableProps> = ({ title, data, description, toolbar, tableHeaders }) => (
+  <DataTable rows={data} headers={tableHeaders} size="short">
+    {({ rows, headers, getHeaderProps, getRowProps, getTableProps, getTableContainerProps }) => (
+      <TableContainer
+        className={styles.tableContainer}
+        title={title}
+        description={description}
+        {...getTableContainerProps()}
+      >
+        {toolbar}
+        <Table {...getTableProps()} isSortable useZebraStyles>
+          <colgroup className={styles.columns}>
+            <col span={1} />
+            <col span={1} />
+            <col span={1} />
+          </colgroup>
+          <TableHead>
+            <TableRow>
+              {headers.map((header) => (
+                <TableHeader key={header.key} {...getHeaderProps({ header })} isSortable>
+                  {header.header}
+                </TableHeader>
+              ))}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {rows.map((row, i) => (
+              <TypedTableRow key={row.id} interpretation={data[i]?.interpretation} {...getRowProps({ row })}>
```

Comment:
I think things that live in `helpers` that are only used once should just be moved to the file where they are used. I would rather `TypedTableRow` be defined in this same file.

Reply:
I agree. I can fix that in another PR.